src/main_loop.rs

```diff
@@ -136,8 +139,8 @@ impl MainLoop {
                     autoplay: self.autoplay,
                     name: self.spotifyd_state.device_name.clone(),
                     device_type: self.device_type,
-                    volume: self.initial_volume.unwrap_or_else(|| mixer.volume()),
-                    volume_ctrl: self.volume_ctrl.clone(),
+                    initial_volume: Some(self.initial_volume.unwrap_or_else(|| mixer.volume())),
```

Comment:
Now, this has become an `Option`, we should be able to safely pass just `self.initial_volume`, since `Spirc` is going to call the mixer itself anyway.

Reply:
👍🏾